roles/edpm_libvirt/tasks/install.yml

```diff
@@ -1,4 +1,11 @@
 ---
+# TODO(arxcruz): Remove this once libvirt 10 is available on centos stream
+- name: Add master repository containing libvirt 10
+  shell:
+    cmd: |
+      dnf config-manager --add-repo https://trunk.rdoproject.org/centos9-master/delorean-deps-test.repo && dnf update -y
```

Comment:
I don't think that we should add a fix related to centos here in this repo.
Can we do this in install_yamls or ci-framework instead?

Reply:
ya this should not be done in the production code this is a tempeory ci issue.

also I'm not sure that the rdo project is the correct place to pull this form 
if we wanted newer libvirt we shoudl use the virt-preview corp repos

we don't maintain libvirt in rdo.
it is not the upstream of what lands in our downstream product
libvirt comes form rhel directly.
OSP had a libvirt package in osp 13 but not in 16 or 17